### DISCUSSION TITLE
What is providetag id?

### DISCUSSION THREAD

**Discussion by tytom2003** [reduxjs/redux-toolkit#4663] [General]
why providetag will using id? such as, [{type: 'Post', id}]. what is id? The id is a virtual unique number for cach each record like sql primary key 
? 
i read website provietag still not understand providetag id.
Thank you very much.

**Comment by markerikson** [comment]
See https://redux.js.org/tutorials/essentials/part-8-rtk-query-advanced#invalidating-specific-items

**Reply by tytom2003** [reply]
The id is make data invalidates for mutation? The id for each cache data?  [{ type: "Post", id }] is specific cache data?
